src/Halogen/Driver.purs

```diff
@@ -131,7 +131,7 @@ runUI c s element = _.driver <$> do
         when (isJust rp) $ render ref
         putVar rpRef Nothing
         q
-      HaltHF -> empty
+      HaltHF msg -> throwError $ error msg
```

Comment:
I say that because I'm pretty sure this gets triggered in SlamData. It can happen even if you have a long running request, and you remove the component before the request completes. It's not necessarily a failure condition, just a timing issue.